cmd/multirootca/ca.go

```diff
@@ -75,6 +81,13 @@ func main() {
 			whitelists[label] = root.ACL
 		}
 		log.Info("loaded signer ", label)
+
+		if root.RootCA != nil {
+			log.Info("loaded root CA ", label)
+			mrBundler.AddRoot(root.RootCA)
```

Comment:
Log after you've loaded the certificate.

Reply:
Good catch indeed